src/julia/install.sh

```diff
@@ -1,10 +1,42 @@
-#!/bin/sh
-set -e
+#!/bin/bash
```

Comment:
By changing from sh to bash, it is no longer possible to run in environments where bash is not installed, but because sh does not support arrays, the USERNAME snippet, which is often used in other Features, could not be used.